src/app.js

```diff
@@ -1 +1,47 @@
+/* eslint-disable no-console */
 'use strict';
+
+const { terminal } = require('./terminal');
+const fs = require('fs');
+const path = require('path');
+
+const promptCopyFile = () => {
+  terminal.question(
+    'What must be copied? (Example: "cp file.txt file-copy.txt"): ',
+    (input) => {
+      const [ command, source, dest ] = input.split(' ');
```

Comment:
Read the task carefully: your app must work from the command line.
Example of usage:
`node app.js file.txt file-copy.txt`